docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# The first instance will automatically bootstrap the cluster.
+# Scale this tier and each additional instance will automatically
+# self-configure as a member of the cluster
+couchbase:
+    image: autopilotpattern/couchbase:latest # enterprise-4.0.0-r1
```

Comment:
I've tagged this PR as `enterprise-4.0.0-r2` for now.

I think the only trouble with the tag is that we're stuffing the Couchbase version along with our own version number. If I were to change it at all, I'd say ditch the Couchbase version and just let people get that from the Dockerfile. But I don't feel strongly enough about it.